### PR TITLE
49 dolphins returned

### PR DESCRIPTION
**Pull request by vikiival**
- 49 sort is back

Thank you for your contribution to the KodaDot NFT gallery, 
we really appreciate your contribution!

animal-dolphins-have-returned-italy-meanwhile-scotland

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### Before submitting this PR, please make sure:
- [x] Your code builds clean without any erros or warnigns
- [x] You've posted screenshot of demonstrated change in this PR
- [x] Merged recent default branch, **main** and you have no conflicts
- [x] Didn't break any original functionality 

### Optional
- [ ] You've tested it on mobile
- [ ] Are there any edge cases? Name if any 

Screenshot 2021-10-04 at 16 27 44

### CODE REVIEW

File: src/components/rmrk/Gallery/Search/SearchBar.vue [hunk 1]
```diff
@@ -38,83 +38,114 @@
...
-    // this.replaceUrl(value)
+    this.replaceUrl(value, 'sort')
     return value
   }
 
   @Emit('update:search')
   @Debounce(400)
-  updateSearch(value: string) {
+  updateSearch(value: string): string {
     console.log('Debounced', value)
```
Comment (roiLeo) [inline src/components/rmrk/Gallery/Search/SearchBar.vue:118]:
```suggestion
```

File: src/components/rmrk/Gallery/Search/SearchBar.vue [hunk 2]
```diff
@@ -38,83 +38,114 @@
...
+  updateType(value: string): string {
     this.replaceUrl(value, 'type')
     return value
   }
 
   @Emit('update:sortBy')
   @Debounce(400)
-  updateSortBy(value: SortBy) {
+  updateSortBy(value: string): string {
     console.log('Debounced', value)
```
Comment (roiLeo) [inline src/components/rmrk/Gallery/Search/SearchBar.vue:110]:
```suggestion

```

**Review by roiLeo — APPROVED** [review]
Nice addition, will look into "listed" filter for series.
I would just remove `console.log` & resolve conflicts.

**Comment by vikiival** [comment]
Thanks for the review @roiLeo  ^-^ 
Let's ship it then 🚢

**Pull request merged**